answers/Objects/CorrectMultipleErrorTest.php

```diff
@@ -0,0 +1,15 @@
+<?php
+    class Person 
+    {
+        public $name = "Diana";
+        
+        public function eat()
+        {
+            echo "This is an eat() method.";
+        }
+    }
```

Comment:
Same as the previous PR. I'm not sure why we need the `eat` method in here

Reply:
Please check the rest as I'm not sure also why we need either the method or the property.

Reply:
This is part of the given class definition in the q.md. This is the one that we presented to the learners as sample method and properties. In part 3 is used to let the learner check for errors. This is also the reason why we have this in our answer. Thank you.

![image](https://user-images.githubusercontent.com/25945790/73618866-6cd8a500-4665-11ea-912c-9e439b5fa302.png)